doc/user/content/sql/system-catalog/mz_catalog.md

```diff
@@ -458,6 +458,32 @@ Field        | Type                 | Meaning
 `owner_id`   | [`text`]             | The role ID of the owner of the table. Corresponds to [`mz_roles.id`](/sql/system-catalog/mz_catalog/#mz_roles).
 `privileges` | [`mz_aclitem array`] | The privileges belonging to the table.
 
+### `mz_timezone_abbrevs`
```

Comment:
Do we want to put these views in `mz_internal`, or are we confident they won't change in the future?

Reply:
I don't expect them to change. They are sufficient to mimic postgres and materialize, and are produced by external data. They were designed to be usable by users today. `mz_catalog` seems like the more accessible place to put them.